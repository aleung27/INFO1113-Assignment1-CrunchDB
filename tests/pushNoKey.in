SET a 1
PUSH A 0
PUSH B 0
POP a
PUSH a 9
GET a
