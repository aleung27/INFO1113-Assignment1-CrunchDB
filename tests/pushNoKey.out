> ok

> no such key

> no such key

> 1

> ok

> [9]

